step-automation-packages/step-automation-packages-support/src/main/java/step/automation/packages/AutomationPackageArchive.java

```diff
@@ -36,14 +36,20 @@ public class AutomationPackageArchive implements Closeable {
     public static final List<String> METADATA_FILES = List.of("automation-package.yml", "automation-package.yaml");
 
     private final ClassLoader classLoader;
+    private final File originalFile;
+    private final String originalFileName;
```

Comment:
@jeromecomte 
It is different only in case of temporary files stored from incoming `InputStream`. If we avoid temp files (with changed names), we can remove the `originalFileName`.